### DISCUSSION TITLE
Does Bale even support bot commands?

### DISCUSSION THREAD

**Discussion by okaeiz** [kianahmadian/python-bale-bot#11] [Q&A] [answered]
I want to create a keyboard with different commands. Is it possible using python-bale-bot?
I have done this before in python-telegram bot using CommandHandlers.

**Comment by kian-ahmadian** [comment, marked as answer]
Hey @okaeiz 

**This is not possible with web services and python-bale-bot**, but you can do this manually through bot father (the bot that produces bots in Bale).
